boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090.rst

```diff
@@ -0,0 +1,206 @@
+.. _nrf9160_pca10090:
+
+nRF9160-PCA10090
+################
+
+Overview
+********
+
+Zephyr applications use the nrf9160_pca10090 board configuration
+to run on the nRF9160 Development Kit (PCA10090) hardware. The
+nRF9160 PCA10090 is a single-board development kit for evaluation
+and development on the nRF9160 SiP for LTE-M and NB-IoT. The
+nrf9160_pca10090 board provides support for the Nordic Semiconductor
+nRF9160 ARM Cortex-M33F CPU with ARMv8-M Security Extension and the
+following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
```

Comment:
We should add `TEE` here

Reply:
Right; i added SPU in the table below. Perhaps i could add IDAU here (generic term)??

Reply:
Added IDAU in this table.